app/assets/stylesheets/components/_mobile-filters.scss

```diff
@@ -74,7 +74,8 @@
     .facets__box {
       border-left: 5px solid #b1b4b6;
       padding-left: govuk-spacing(2);
-      margin-bottom: govuk-spacing(3);
+      padding-bottom: 2px;
+      margin-bottom: govuk-spacing(4);
```

Comment:
This feels a bit hacky but it seems like you're having to jump through this hoop due to an oddity with the button component - it's got a margin bottom of zero but a 2px box shadow, which means it always hangs down a little. That's an oddity we've introduced (the Design System button appears to have a default bottom margin of 32px, which would be the 2px plus the DS spacing scale). It might be worth investigating why our button component is like that? Personally I'd be tempted to set the default margin bottom to 2px so it doesn't hang down and we don't have this problem.

Reply:
Yep, it's hacky and I'm not too happy about it, but it seems the simplest option give these issues that are causing it:
- box shadows aren't computed as part of an elements size, so they don't take up any space. In this case, they fall outside the container, making the button appear misaligned with the border on the left. 
- govuk buttons have margins, but gem buttons don't, they're reset to 0 so the button is flush at the end of the container and thus the shadow falls outside.

The 2px of padding that I added is to force a space for the shadow, making the container just long enough so that the button appears to line up fully with the left hand border. 

Another solution would be to fiddle with a specific margin on this specific button that accounts for all these variables, but that seems like an even worse hack.

We could also remove the left hand grey border and padding from the `facets__box`. This border was clearly added to wrap everything and build a relationship with the Filter toggle at the top, so it is useful. The question then is whether it's more important to have this useful indicator or to have the button shadow line up with it. I know which I'd choose for the sake of simplicity, but it was a difference of opinion on that which made me do this hack in the first place.

Personally I'd solve the whole thing by rethinking the use of box-shadows to simulate borders,  but that's not my call.

This is a lot of thinking for such a tiny issue! I'm therefore inclined to leave this 'fix' alone.  🫣

Reply:
Okay, let's leave as is for now. I've asked the Design System team about this bottom box shadow, interested to see what they say.